documentation/generic-transformer.md

```diff
@@ -1,14 +1,9 @@
 # Generic Transformer
-The `contractWatcher` command is a built-in generic contract watcher. It can watch any and all events for a given contract provided the contract's ABI is available.
-It also provides some state variable coverage by automating polling of public methods, with some restrictions:
-1. The method must have 2 or less arguments
-1. The method's arguments must all be of type address or bytes32 (hash)
-1. The method must return a single value
+The `contractWatcher` command is a built-in generic contract watcher.
+It can watch events for a given contract provided the contract's ABI is available.
 
 This command requires the contract ABI be available on Etherscan if it is not provided in the config file by the user.
-
-If method polling is turned on we require an archival node at the ETH ipc endpoint in our config.
-Otherwise we only need to connect to a full node.
+Optionally, pass a `--etherscan-api-key` (`-k`) flag to include your Etherscan API key when running this command, if looking up multiple ABIs.
```

Comment:
Any chance this could be consolidated anywhere? I thought something was busted when I read this for the third time.

Reply:
Definitely down for some consolidation, but I think this is only duplicated bw the CLI help section + README? I'd personally be comfortable have information duplicated across those two places, but am open to reconsidering (or lmk if I'm straight up missing something)

Reply:
My concern is with three places it's likely to diverge. It would be great if we could write it in one place and extract it to all three places it's read. But that's bigger than this story.

Reply:
but what's the third place?